.travis.yml

```diff
@@ -0,0 +1,18 @@
+script:
+- echo "hello world"
+- docker build --target development --tag todo-app:dev .
+- docker build --target production --tag todo-app:prod .
```

Comment:
Worth noting that there's no major gain from building these images here; we'd usually stick to building when we're actually going to use the image e.g. when we're ready to push the image to a repository & deploy it to a testing environment which we'll add in as part of module 8

Reply:
Removed the un-necessary builds